cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,41 @@
-/// <reference types='cypress' />
-/// <reference types='../support' />
+import ArticlePageObject from '../support/pages/article.pageObject';
+
+const ArticlePage = new ArticlePageObject();
 
 describe('Article', () => {
-  before(() => {
+  let user;
+  let article;
 
+  before(() => {
+    cy.task('db:clear');
   });
 
   beforeEach(() => {
-    cy.task('db:clear');
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+      cy.login(user);
+      cy.visit(`/#/editor`);
+    });
+    cy.task('generateArticle').then((data) => {
+      article = data;
+    });
```

Comment:
The generated article data is assigned to the 'article' variable, but the article is not actually created in the system before the tests run. Consider creating the article in the system here to ensure it exists before each test.